test/functional/tests/io_class/test_io_class_occupancy_load.py

```diff
@@ -0,0 +1,181 @@
+#
+# Copyright(c) 2020 Intel Corporation
+# SPDX-License-Identifier: BSD-3-Clause-Clear
+#
+
+from collections import namedtuple
+from math import isclose
+
+import pytest
+
+from .io_class_common import *
+from api.cas.cache_config import CacheMode, CacheLineSize
+from api.cas.casadm_params import OutputFormat
+from api.cas.ioclass_config import IoClass
+from storage_devices.device import Device
+from storage_devices.disk import DiskType, DiskTypeSet, DiskTypeLowerThan
+from test_tools import fs_utils
+from test_tools.disk_utils import Filesystem
+from test_utils.os_utils import sync, Udev
+
+
+@pytest.mark.require_disk("cache", DiskTypeSet([DiskType.optane, DiskType.nand]))
+@pytest.mark.require_disk("core", DiskTypeLowerThan("cache"))
+@pytest.mark.parametrizex("cache_line_size", CacheLineSize)
+def test_ioclass_occuppancy_load(cache_line_size):
+    """
+        title: Load cache with occupancy limit specified
+        description: |
+          Start cache in WB, add ioclasses with a occupancy limit. Populate cache
+          and stop it without a flush. Load cache and verify if occupancy limits
+          are loaded correctly and if each part has assigned apropriate number of
+          dirty blocks.
+        pass_criteria:
+          - Occupancy thresholds have correct values for each ioclass after load
+    """
+    with TestRun.step("Prepare disks"):
```

Comment:
I'd rather call this step "Prepare CAS devices" or something like that, maybe "Start cache in Write-Back mode and add core device". Please do this in all tests.

Reply:
I went with `Prepare CAS devices`